test/fixtures/uncaught-exceptions/callbackify2.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// Used to test arguments to `uncaughtException` from a callback
+
+const assert = require('assert');
+const { callbackify } = require('util');
+
+{
+  const sentinel = new Error(__filename);
+  process.once('uncaughtException',(err) => {
+    assert.strictEqual(err, sentinel);
+    console.log(err.message);
```

Comment:
In that case, leaving a comment that it’s part of the test might be a good idea :)